city_scrapers/spiders/det_library_commission.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import urllib.parse
+from dateutil.parser import parse
+import scrapy
+from city_scrapers.spider import Spider
+
+
+class DetLibraryCommissionSpider(Spider):
+    name = 'det_library_commission'
+    agency_id = 'Detroit Public Library'
+    timezone = 'America/Detroit'
+    allowed_domains = ['detroitpubliclibrary.org']
+    start_urls = ['https://detroitpubliclibrary.org/about/commission']
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the Event Schema
+        <https://city-bureau.github.io/city-scrapers/06_event_schema.html>.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+
+        yield from self._generate_requests(response)
+
+    def _generate_requests(self, response):
+        anchor_xpath = '//div[contains(@class, "card-details")]/a'
+        anchors = response.xpath(anchor_xpath)
+        for a in anchors:
+            yield response.follow(a, self._parse_item)
+
+    def _parse_item(self, response):
+        name = self._parse_name(response)
+        location = self._get_location(response)
+        classification = self._parse_classification(name)
+        date, start_time, end_time = self._parse_date_and_times(response)
+        data = {
+            '_type': 'event',
+            'name': name,
+            'event_description': '',
+            'classification': classification,
+            'start': {
+                'date': date,
+                'time': start_time,
+                'note': '',
+            },
+            'end': {
+                'date': date,
+                'time': end_time,
+                'note': '',
+            },
+            'all_day': False,
+            'location': location,
+            'documents': [],
```

Comment:
It would be great to pull the minutes here and create the old meetings from the (limited) details in the minutes if you think it's doable. It's not a problem if you don't think there's enough to go off of there though